modules/uninstall-cluster-logging-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-cluster-logging-operator_{context}"]
+= Uninstalling the {logging}
+
+You can stop log aggregation by deleting the {clo} and the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the *Administrator* perspective of the {product-title} web console.
+
+.Procedure
+
+. Navigate to the *Administration* -> *Custom Resource Definitions* page. Click *ClusterLogging*.
```

Comment:
```suggestion
. Go to the *Administration* -> *Custom Resource Definitions* page, and click *ClusterLogging*.
```
- [The ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#n) says to avoid using "navigate" and to use alternatives such as "go to" when you can.
- I would combine these two sentences into one because these two actions are presented as a single step. If you keep them as separate sentences, it feels to me like they should be presented as separate steps.